tests/test_importer_utils.py

```diff
@@ -91,8 +91,7 @@ def test_get_text_inside_brackets_nested_two(self):
 
     def test_get_text_inside_brackets_twice(self):
         text = "pre (foo) mid (bar) post"
-        output = "foo"
-        self.assertEqual(utils.get_text_inside_brackets(text), output)
+        self.assertRaises(ValueError, utils.get_text_inside_brackets, text)
```

Comment:
Could you also add a test for `"pre (foo"` I believe it will not give the expected result (either `ValueError` or "foo").

This comes about because find returns `-1` if not found.
You can get around this (and similar problems with strings such as `"foo )bar( cat"` and `"foo) bar (cat)"`) by:
```python
first_bracket_index = text.find('(')
last_bracket_index = text[first_bracket_index:].rfind(')')
if last_bracket_index < 0 or text[:first_bracket_index].rfind(')') > 0:
     raise ValueError('Unmatched brackets encountered')
result = text[first_bracket_index + 1:first_bracket_index+last_bracket_index]
```